variables.md

```diff
@@ -277,3 +277,5 @@ PUBLIC_CLOUD_USER | string | "" | The public cloud instance system user.
 PUBLIC_CLOUD_XEN | boolean | false | Indicates if this is a Xen test run.
 PUBLIC_CLOUD_TERRAFORM_FILE | string | "" | If defined, use this terraform file (from the `data/` directory) instead the CSP default
 TERRAFORM_TIMEOUT | integer | 1800 | Set timeout for terraform actions
+PUBLIC_CLOUD_EXISTING_ADDRESS | string | "" | If defined, no instance will be created and this IP will be used to connect to
+PUBLIC_CLOUD_EXISTING_SSH_KEY | string | "" | The `~/.ssh/id_rsa` existing key allowed by `PUBLIC_CLOUD_EXISTING_ADDRESS` instance
```

Comment:
I think we should handle this as a `ssh-provider` like `img-proof` is doing it, see: https://img-proof.readthedocs.io/en/latest/index.html#cloud-framework-classes
```
The SSH class is generic and can be used for any accessible instance that is running. There are no credentials required except the instance needs the proper SSH User and SSH Key configured for access.
```

Reply:
I'm closing this as a duplicity of [your other suggestion](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14984#discussion_r884888336) - let's continue over there.